plugins/v8-xhr.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Chrome/V8 XHR Post plugin
+ *
+ * Allows raven-js to send log entries using XHR and CORS for
+ * environments where the Origin and Referer headers are missing (e.g Chrome plugins).
+ *
+ * Usage: Raven.config('https+post://...');
+ * */
+
+;(function(window, Raven, undefined){
+    'use strict';
+    var V8Transport = window.V8Transport = {
+        setup: function(dsn, triggerEvent){
+            if(!this.hasCORS() && window.console && console.error){
+                console.error('This browser lacks support for CORS. Falling back to the default transport');
```

Comment:
Shouldn't the `window.console && console.error` check be only around this `console.error()` call?